go.mod

```diff
@@ -1,9 +1,17 @@
 module github.com/bmaupin/go-epub
 
-go 1.15
+go 1.20
```

Comment:
Regarding versions, while this PR was waiting review go 1.21 came out and 1.20 is the oldest version being maintained: https://endoflife.date/go